platform-sdk/swirlds-merkledb/src/test/java/com/swirlds/merkledb/collections/AbstractLongListTest.java

```diff
@@ -53,6 +55,19 @@ protected AbstractLongList<?> createLongList() {
         return new LongListHeap();
     }
 
+    protected LongListDisk createLongListDisk(final Configuration configuration) {
+        return new LongListDisk(configuration);
+    }
+
+    /**
+     * Provides a LongList implementation intended for writing data.
+     * This may be used in tests to verify that lists created by this "writer" can
+     * be successfully read back by other implementations.
+     *
+     * @return a LongList for writing operations
+     */
+    protected abstract AbstractLongList<?> createWritingLongList();
```

Comment:
For some reason, `LongListDiskTest` does not extend this class, `AbstractLongListTest`. That's why some test cases may be missing for LongListDisk, and some other test cases can be missing for LongListHeap/OffHeap

Reply:
Yes. Do you mean that it would be good to revisit `LongListDiskTest` so it extends `AbstractLongListTest`?